Mathlib/GroupTheory/Perm/Centralizer.lean

```diff
@@ -0,0 +1,2379 @@
+/-
+Copyright (c) 2023 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+
+! This file was ported from Lean 3 source module conj_class_count
```

Comment:
But it was never part of mathlib3, right? So I think there is not so much value in registering this in the file header.

Reply:
And maybe this also means that the `#align` statements are not needed?

Reply:
It was provided by mathport — I'll delete them.